hazelcast/src/main/java/com/hazelcast/nio/serialization/PortableReader.java

```diff
@@ -24,6 +24,8 @@
 /**
  * Provides a mean of reading portable fields from a binary in form of java primitives
  * arrays of java primitives , nested portable fields and array of portable fields.
+ * <p/>
+ * fieldName used in most of the methods may contain nested fields, like body.brain.iq
```

Comment:
Sure. Earlier it was not clear to me that it may be a nested path. Could you rephrase it to a more formal sentence?